docs/aggregate-apache-logs-with-fluentd-and-minio.md

```diff
@@ -1,10 +1,10 @@
-# Aggregate Apache Logs with fluentd plugin for Minio [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+# Aggregate Apache Logs with fluentd plugin for MinIO [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
```

Comment:
`https://slack.minio.io/slack?type=svg` should be `https://slack.min.io/slack?type=svg` and `https://slack.minio.io` should be `https://slack.min.io`